run_client_tests

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+echo "Running client tests..."
+
+pytest client/tests
+
+rm test.db
```

Comment:
Could you add it as part of the test itself? Then pytest can be used directly instead of adding yet another script

Reply:
I like the idea of having that in a script but I guess it could be a more clean solution if I did everything in pytest.